bob/bob.go

```diff
@@ -195,3 +201,13 @@ func (b *B) read() (err error) {
 
 	return nil
 }
+
+// keepWhitelistEnv will keep whitelisted env variables
+// from local host
+func (b *B) keepWhitelistEnv() {
+	for _, envKey := range bobutil.EnvWhitelist {
+		if value, exists := os.LookupEnv(envKey); exists {
+			b.env = append(b.env, envKey+"="+value)
```

Comment:
Needs a check if the whitelisted env is already present in b.env. Otherwise you can't overwrite whitelisted env vars.

Reply:
Added test for it, it looks like the order in which they are added it makes this possible so no change is required here